integration_tests/src/test/resources/decimals.csv

```diff
@@ -0,0 +1,14 @@
+"number"
+-1
+0
+1.
+0.12
+.12
++.12
+-.12
+1
+1.01
+12.34
+12.3456
```

Comment:
Could we have a few more numbers to see if Spark is rounding when the scale is larger than expected, or if it is truncating?